packages/office-ui-fabric-react/src/components/Stack/StackItem/StackItem.types.ts

```diff
@@ -16,7 +16,7 @@ export interface IStackItemProps extends IStackItemSlots, IStyleableComponentPro
   /**
    * Defines how to render the StackItem.
    */
-  as?: string | React.ReactType<IStackItemProps>;
+  as?: React.ReactType<React.HTMLAttributes<HTMLElement>>;
```

Comment:
@khmakoto @kkjeer  I don't see this prop being used. Should it be removed?

Reply:
I think it may be removed if it's not being used. Better to remove now that it has just been promoted and add it later if needed than to try to delete it later.

Reply:
by the way, when do we have an updated version with the issue fixed

Reply:
Which issue are you talking about? This PR was released with [v6.141.0](https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v6.141.0).